src/content/docs/dropins-b2b/purchase-order/events.mdx

```diff
@@ -0,0 +1,658 @@
+---
+title: Purchase Order Data & Events
+description: Learn about the events used by the Purchase Order and the data available within the events.
+sidebar:
+  label: Events
+  order: 5
+---
+
+import { Aside } from '@astrojs/starlight/components';
+import TableWrapper from '@components/TableWrapper.astro';
+
+The **Purchase Order** drop-in uses the [event bus](/sdk/reference/events) to emit and listen to events for communication between drop-ins and external integrations.
+
+<div style="background-color: var(--sl-color-blue-low); border-left: 4px solid var(--sl-color-blue); padding: 0.75rem 1rem; border-radius: 0.25rem; margin: 1rem 0;">
+<strong>Version: 1.0.0-beta2</strong>
+</div>
+
+## Events reference
+
+{/* EVENTS_TABLE_START */}
+<TableWrapper nowrap={[0, 1]}>
+
+| Event | Direction | Description |
+|-------|-----------|-------------|
+| [order/data](#orderdata-emits) | Emits | Emitted when data is available or changes. |
+| [purchase-order/error](#purchase-ordererror-emits) | Emits | Emitted when an error occurs. |
+| [purchase-order/placed](#purchase-orderplaced-emits) | Emits | Emitted when an order is placed. |
+| [auth/permissions](#authpermissions-listens) | Listens | Fired by Auth (`auth`) when permissions are updated. |
+| [purchase-order/data](#purchase-orderdata-emits-and-listens) | Emits and listens | Triggered when data is available or changes. |
+| [purchase-order/refresh](#purchase-orderrefresh-emits-and-listens) | Emits and listens | Emitted and consumed for internal and external communication. |
+
+</TableWrapper>
+{/* EVENTS_TABLE_END */}
+
+## Event details
+
+The following sections provide detailed information about each event, including its direction, event payload, and usage examples.
+
+
+### `auth/permissions` (listens)
+
+Fired by Auth (`auth`) when permissions are updated.
+
+#### Event payload
+
+```typescript
+{
+  admin?: boolean;
+  [key: string]: boolean | undefined;
+}
+```
+
```

Comment:
I've been ignoring the extraneous blank lines up to here, but this is too many. Reduce the amount, please.